fairlearn/reductions/grid_search/grid_search.py

```diff
@@ -107,18 +106,18 @@ def __init__(self,
         self.grid_limit = float(grid_limit)
         self.grid = grid
 
-    def fit(self, X, y, aux_data=None, **kwargs):
+    def fit(self, X, y, **kwargs):
         if X is None:
             raise ValueError(self._MESSAGE_X_NONE)
 
         if y is None:
             raise ValueError(self._MESSAGE_Y_NONE)
 
-        if aux_data is None:
-            raise RuntimeError("Must specify aux_data (for now)")
+        if _KW_GROUP_MEMBERSHIP not in kwargs:
+            raise RuntimeError("Must specify {0} (for now)".format(_KW_GROUP_MEMBERSHIP))
 
         # Extract the target attribute
-        A = self._make_vector(aux_data, "aux_data")
+        A = self._make_vector(kwargs[_KW_GROUP_MEMBERSHIP], _KW_GROUP_MEMBERSHIP)
```

Comment:
Going back to `A` will only confuse people.

Reply:
Agreed. i'll fix this specific instance (A -> sensitive), and leave the additional fixing for a future PR... since this PR is blocking too many files.